makeScanFileII.py

```diff
@@ -57,8 +57,8 @@ def doMakeScanFile(ConfigInfo):
     FillfromDip = df['fill'][0]
 
     if (Fill != str(FillfromDip)):
-            print("Mismatch between fill info from dip and from config file. Exit program.")
-            sys.exit(1)
+            print("Mismatch between fill info from dip and from config file.")
+            #sys.exit(1)
```

Comment:
#sys.exit(1) --> 
raw_input("Mismatch between fill info from dip and from config file; Really Continue?")